roles/ovirt-datacenter-cleanup/README.md

```diff
@@ -17,6 +26,9 @@ Role Variables
 |--------------------------|-----------------------|--------------------------------------|
 | data_center_name         | UNDEF                 | Name of the data center.             |
 | format_storages          | false                 | Whether role should format storages when removing them. |
+| hosted_engine            | false                 | Specify whether we datacenter to cleanup contains a VM which is serving an OVirt hosted engine installation.|
```

Comment:
Maybe `hosted_engine_remove` would be more reasonable name?

Reply:
Hmm. I find hosted_engine_remove naming more confusing.
If hosted_engine_remove == true I would expect that the role will actually remove the hosted_engine VM.

Thought hosted_engine name for parameter just implies that the datacenter we are handling contains hosted_engine VM. This VM will not we removed of course.